index.bs

```diff
@@ -5692,14 +5692,27 @@ useful when there is instance data to track.
 The class would be used in code like:
 
 <pre><code class="lang-javascript">
+  function textDecoderStream() {
+    const td = new TextDecoder();
+    return new TransformStream({
+      transform(chunk, controller) {
+        controller.enqueue(td.decode(chunk));
+      }
+    });
+  }
+
   const data = { userName, displayName, icon, date };
   const ts = new TransformStream(new LipFuzzTransformer(data));
+  const { url } = fetchEvent.request;
```

Comment:
Might be better not to have a `url` intermediate value, but just extract the value directly in the call to fetch().